src/main/java/io/javalin/plugin/openapi/annotations/AnnotationApi.kt

```diff
@@ -0,0 +1,75 @@
+package io.javalin.plugin.openapi.annotations
+
+import kotlin.reflect.KClass
+
+/**
+ * Provide metadata for the generation of the open api documentation to the annotated Handler.
+ */
+@Target(AnnotationTarget.FUNCTION)
+annotation class OpenApi(
+        val summary: String = NULL_STRING,
+        val description: String = NULL_STRING,
+        val operationId: String = NULL_STRING,
+        val deprecated: Boolean = false,
+        val tags: Array<String> = [],
+        val cookies: Array<OpenApiParam> = [],
+        val headers: Array<OpenApiParam> = [],
+        val pathParams: Array<OpenApiParam> = [],
+        val queryParams: Array<OpenApiParam> = [],
+        val requestBodies: Array<OpenApiRequestBody> = [],
+        val fileUploads: Array<OpenApiFileUpload> = [],
+        val responses: Array<OpenApiResponse> = []
+)
+
+@Target()
+annotation class OpenApiResponse(
+        val status: String,
+
+        val returnType: KClass<*> = NULL_CLASS::class,
+
+        val contentType: String = ContentType.AUTODETECT,
+
+        /** Whenever the returnType should be wrapped in an array */
+        val isArray: Boolean = false,
+
+        val description: String = NULL_STRING
+)
```

Comment:
Why is the property spacing different in `OpenApiResponse` (space between each property) compared to th eother annotation classes?

Reply:
Just an error. I'm going to fix it,